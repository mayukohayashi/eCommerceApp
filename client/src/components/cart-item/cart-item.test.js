import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cart-item.component';

it('should render cart-item component', () => {
  const mockItem = {
    imageUrl: 'www.testImage.com',
    price: 900,
    name: 'Jackets',
    quantity: 2
  };
  expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
});
