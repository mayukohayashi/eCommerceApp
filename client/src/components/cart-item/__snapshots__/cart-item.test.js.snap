// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render cart-item component 1`] = `ShallowWrapper {}`;
